kudu/src/main/antlr4/KuduSQLExpression.g4

```diff
@@ -0,0 +1,154 @@
+grammar KuduSQLExpression;
```

Comment:
Apache license header?

Reply:
Thanks for catching this. Will add the license. It looks like Antlr4 has issues with comments starting with a double asterisk. /**. Will add the license header. Hope the license check does not fail if it looks for a double asterisk at the beginning.